dc_signup_form/tests/test_forms.py

```diff
@@ -1,36 +1,59 @@
 from django.test import TestCase
 from dc_signup_form.forms import (
     ElectionRemindersSignupForm, MailingListSignupForm)
+from dc_signup_form.constants import (
+    MAILING_LIST_FORM_PREFIX,
+    ELECTION_REMINDERS_FORM_PREFIX
+)
+
+def add_data_prefix(prefix, data):
+    return {
+        "{}-{}".format(prefix, k): v
+        for k, v in data.items()
+        if not str(v).startswith(k)
+    }
 
 
-class TestForms(TestCase):
 
+class TestForms(TestCase):
     def test_mailing_list_form_valid(self):
-        form = MailingListSignupForm(data={
+        data = add_data_prefix(MAILING_LIST_FORM_PREFIX, {
             'full_name': 'Chad Fernandez',
             'email': 'chad.fernandez@democracyclub.org.uk',
             'source_url': 'http://foo.bar/baz',
             'main_list': True,
         })
+
+        form = MailingListSignupForm(
+            data,
+            prefix=MAILING_LIST_FORM_PREFIX,
```

Comment:
Sorry.. just spotted one more thing: Now that we're setting `prefix` in the form classes, all the calls in this test class where we're manually passing a prefix into the form constructors can be removed.